### PR TITLE
Add support for php5.3 and update README

### PR DESCRIPTION
**Pull request by shinigamicorei7**
En el README agrego información relevante del repositorio.

[Build Status](https://travis-ci.org/ecoal95/php-router) [Latest Stable Version](https://packagist.org/packages/ecoal95/php-router) [Total Downloads](https://packagist.org/packages/ecoal95/php-router) [Latest Unstable Version](https://packagist.org/packages/ecoal95/php-router) [License](https://packagist.org/packages/ecoal95/php-router)

Y agrego soporte para la versión 5.3 de PHP.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+.idea
```
Comment (emilio) [inline .gitignore:3]: This is editor-specific, consider adding it to your global gitignore (https://help.github.com/articles/ignoring-files/)

File: examples/example.php [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-$router->add('/hola/([a-zA-Z]+)', function ($name) {
-    echo sprintf('<h1>Hola %s</h1>', $name);
-});
-
-$router->add('/.*', function () {
-    header($_SERVER["SERVER_PROTOCOL"] . " 404 Not Found");
-    echo '<h1>404 - El sitio solicitado no existe</h1>';
-});
-
-$router->route();
```
Comment (emilio) [inline examples/example.php:15]: Why did you remove this?

**Comment by emilio** [comment]
Thanks for the changes! Consider squashing the commits the next time because if you don't the repo will be bloated :P

I submitted your changes (with partial modifications) to master in https://github.com/ecoal95/php-router/commit/4871c5c565a63f15453e0eceb778b957f0f33f3b

I left a few comments in the commits, if there's a good reason to remove the examples directory, tell me.

Thanks again! :)

**Comment by shinigamicorei7** [comment]
Sorry'm fairly new to github, next time will be better :cry: :+1:

**Comment by emilio** [comment]
Don't worry!

The changes are more than welcome :))
